packages/session-replay-browser/test/integration/sampling.test.ts

```diff
@@ -117,10 +117,8 @@ describe('module level integration', () => {
           // Ensure that sample rate matches what's passed in the options
```

Comment:
Nit: can you update the test label to show that its on purpose now that replays aren't captured?